### DISCUSSION TITLE
SVGEDITOR with Java Webapp

### DISCUSSION THREAD

**Discussion by NK121190** [SVG-Edit/svgedit#809] [General]
Hi,
   I have integrated the SVG editor in my web app and placed version 5 inside my project. I am calling directly the parent HTML file(Svgeditor.html)  using jquery modal. On click of saving, I need to get the SVG source or the blob of the image in return from the editor. is it possible? if so, kindly post your inputs to achieve the same.

Thanks in Advance.

**Comment by jfhenon** [comment]
Hello, 
would you be able to try using V7? 
if not, could you explain the reason why?
Thanks

**Reply by NK121190** [reply]
Thanks for the reply @jfhenon. It was already integrated with the web app in 2019 itself. The web app is using struts framework using Websphere server. My work is to get and set the final source of the image that we are editing as well as that going to edit, these needs to be handled in the normal javascript. The svg and jpeg final encoded data or blob which passed to the <a> tag for the download those i need to get back from the editor that opened in the jquery modal.

Suggest the ways to get those from the editor.

Thanks ,
NK

**Comment by NK121190** [comment]
Exporting Image by JPEG or PNG, when the Image has Text, It is giving Undefined on Export. The exception happens in the replace function present in the canvg.js file.
